include/istio/control/http/report_data.h

```diff
@@ -47,6 +47,10 @@ class ReportData {
   // Get destination ip/port.
   virtual bool GetDestinationIpPort(std::string* ip, int* port) const = 0;
 
+  // Get rbac permissive mode policy attributes.
+  virtual void GetRBACPermissiveAttributes(std::string* resp_code,
```

Comment:
how about define

struct RbacReportInfo {

};

and a get function with

bool GetRbacReportInfo(  struct * info);

Reply:
thanks, done.